src/autogluon/bench/runbenchmark.py

```diff
@@ -191,8 +194,13 @@ def get_job_status(
 
     for job_id in job_ids:
         response = batch_client.describe_jobs(jobs=[job_id])
-        job = response["jobs"][0]
-        status_dict[job_id] = job["status"]
+        job_detail = response["jobs"][0]
+
+        # Check if the job is an array job
```

Comment:
If the job which is submitted is more than 1 than is it not going to be an Array job by default?

Reply:
Array job is a single parent job. Technically at this moment, `job_ids` array will contain only one object.